products/sle/main.pm

```diff
@@ -74,6 +74,22 @@ sub is_bridged_networking {
     return $ret;
 }
 
+sub default_desktop {
+    my $defdesktop = 0;
```

Comment:
Could get rid of this variable and return the result instead of assigning value. This will allow to get rid of else statements, but it's matter of taste of course. Other than that LGTM.